vector/v.in.ogr/main.c

```diff
@@ -1175,13 +1181,23 @@ int main(int argc, char *argv[])
 			      Ogr_fieldname, OFTIntegerListlength);
 		}
 		else {
-		    G_warning(_("Column type (Ogr_ftype: %d) not supported (Ogr_fieldname: %s)"),
-			      Ogr_ftype, Ogr_fieldname);
-		    buf[0] = 0;
-		    col_info[i_out].type = G_store(buf);
-		}
-		G_free(Ogr_fieldname);
-	    }
+                /* handle columns of unsupported data type */
+                G_warning(_("Column type (Ogr_ftype: %d) not supported (Ogr_fieldname: %s)\n"),
+                          Ogr_ftype, Ogr_fieldname);
+                if (flag.drop->answer) {
```

Comment:
As above, the drop flag does not make sense: unsupported column types can not be transferred in any reasonable way.